hs_core/management/commands/name_norm_test.py

```diff
@@ -0,0 +1,558 @@
+    u"Scott Compton",
+    u"Scott Hynek",
+    u"S.D. Sebestyen",
+    u"Seyfried, Mark S.",
+    u"Seyfried, M.S.",
+    u" Shale Hills Critical Zone Observatory",
+    u"Shale Hills Critical Zone Observatory",
+    u"Shale Hills CZO",
+    u"Sharon Billings",
+    u"Sharon Cantrell",
+    u"S. Havens",
+    u"Shawn Alexander Pedron",
+    u"Sheila F. Murphy",
+    u"Shirley Papuga",
+    u"Shi, Yuning",
+    u"Shrestha, Rupesh",
+    u"Shuttleworth, William J.",
+    u"Silver, Whendee L.",
+    u"Singha, Kamini",
+    u"Slingerland, Rudy",
+    u"S.L. Johnson",
+    u"Smith, Lauren",
+    u"Southern Sierra Critical Zone Observatory",
+    u"Spaete, L.P.",
+    u"Stacy, E.",
+    u"Stallard, R.F.",
+    u"Stanbery, C.",
+    u"Steel, Peter",
+    u"Steven Glaser",
+    u"Steven Henning",
+    u"Steven J. Hall",
+    u"Steven S. Van Vactor",
+    u"Stone M.M.",
+    u"Stone, M.M.",
+    u"Stromberg, Mark",
+    u"Stroud Water Research Center",
+    u"Stuemky, M.",
+    u"Sue Brantley",
+    u"Susan Brantley",
+    u"Susan L Brantley",
+    u"Susan L. Brantley",
+    u"Susan Yetter",
+    u"Sutter, Lori",
+    u"Suzanne Anderson",
+    u"Sweeney, B.",
+    u"Szabo, Timea",
+    u"Tang, Qicheng",
+    u"Tess Russo",
+    u"Thomas, Evan",
+    u"Thomas, Evan M.",
+    u"Thompson, A.",
+    u"Timothy Corley",
+    u"Timothy D. Schowalter",
+    u"Todd, Dawson E.",
+    u"Toran, Laura",
+    u"Torres-Sánchez, Angel",
+    u"Torres-Sanchez, Angel J. ",
+    u"T. Parzybok",
+    u"Treffkorn, Jonathan",
+    u"Tsang, Y-P",
+    u"Tyson Lee Swetnam",
+    u"University of Arizona",
+    u"University of California Merced",
+    u"U.S. Army Corps of Engineers (USACE)",
+    u"USDA-ARS",
+    u"USDA ARS Northwest Watershed Research Center",
+    u"USDA-ARS Northwest Watershed Research Center",
+    u"USDA USFS",
+    u"U.S. Department of Agriculture",
+    u"US Geological Survey",
+    u"US Geological Survey Caribbean Water Science Center",
+    u"USGS",
+    u"USGS National Water Information System",
+    u"USGS NWIS",
+    u"Valles Caldera National Preserve",
+    u"Wadsworth, Frank H.",
+    u"Weitzman, Julie",
+    u"West, H.",
+    u"Whendee L. Silver",
+    u"Whendee Silver",
+    u"White, Tim",
+    u"W.H. McDowell",
+    u"William Dietrich",
+    u"William G. McDowell",
+    u"William H McDowell",
+    u"William H. McDowell",
+    u"Williams, Cameron",
+    u"Williams, Jennifer",
+    u"William Wright",
+    u"Will, R.M.",
+    u"W. L. Silver",
+    u"Wong, Christopher S.",
+    u"Xavier Comas",
+    u"Xavier Zapata-Rios",
+    u"Xiande Meng",
+    u"Xiao, Dacheng",
+    u"Xing Hao",
+    u"Xing, Hao Xing",
+    u"Yan Lin",
+    u"Yannis Dialynas",
+    u"Yao Tang",
+    u"Yawen Shen",
+    u"Yuting He,",
+    u"Yu, Xuan",
+    u"Zachary Brecheisen",
+    u"Zhuo Wang",
+    u"Zreda, Marek",
+    u"Z.S. Brecheisen"
+]
+
+
+class Command(BaseCommand):
+    help = "Set resource 'metadata_dirty' and 'bag_modified' AVUs to true"
```

Comment:
Looks like this is left over from copy/paste.  Also, it's not clear to me what this command is doing.  It takes a list of names defined in the command and runs it through normalize_name.  Then it just prints out the normalized name along with the original name.  This looks like it was a test that you can do by eye.  

It's worth setting up a unit test comparing the result with an expectation for this list.

Reply:
What this command does is to demonstrate to Mark that his names don't get mangled any more. Then it can be deleted.

Reply:
Willing to write unit tests once my algorithm is deemed acceptable. 

@sblack-usu The story behind this is that the CZO names got mangled by the current name parser and normalizer. This PR is a response to that. This command demonstrates that the problem is solved by showing what it does for all of the affected names. Then it can be deleted.

Reply:
At a deeper level, I propose that this algorithm can be used to sort out naming on the "my resources" page, where the sort order by name doesn't work because those names are not normalized.